pkg/api/admin/openshiftcluster_convert.go

```diff
@@ -179,6 +179,7 @@ func (c openShiftClusterConverter) ToExternal(oc *api.OpenShiftCluster) interfac
 	}
 
 	if oc.Identity != nil {
+		out.Identity = &ManagedServiceIdentity{}
```

Comment:
You should check if `out.Identity == nil` or this will clobber anything that is already there. This would have been a 100% panic if `out.Identity` were always `nil`. Is it?

Reply:
The `out` object is created from scratch earlier up in this function (L18-75), so we don't need to worry about clobbering any existing Identity properties, `out.Identity` is always `nil` at this point.

Reply:
So are you saying this code would always panic? Your original comment made it seem like it was normally fine and only having a panic in MIWI

Reply:
The if statement this line is contained in, `if oc.Identity != nil`, would only be true for MIWI clusters, but the code within that if block would always panic since `out.Identity` was always nil.